packages/frontend/src/utils/kpi.ts

```diff
@@ -0,0 +1,40 @@
+export const getReachedGoalPercentage = (
+    tvl?: number,
+    lowerBound?: number,
+    upperBound?: number,
+) => {
+    if (
+        tvl === undefined ||
+        lowerBound === undefined ||
+        upperBound === undefined
+    )
+        return 0;
+
+    if (tvl < lowerBound) return 0;
+    if (tvl >= upperBound) return 100;
+
+    const totalRange = upperBound - lowerBound;
+    const reachedRange = tvl - lowerBound;
+    const reachedPercentage = reachedRange / totalRange;
+    return reachedPercentage * 100;
```

Comment:
I was wondering. Since later, when using this function, we're dividing by 100 anyway, can we skip the `* 100` here and the `/ 100` there? But only if it doesn't change too much stuff, this is super minor after all